api/controllers/VisitsController.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * VisitsController
+ *
+ * @description :: Server-side logic for managing visits
+ * @help        :: See http://sailsjs.org/#!/documentation/concepts/Controllers
+ */
+
+module.exports = {
+
+  /**
+   * `VisitsController.create()`
+   */
+  create: function (req, res, next) {
+    var params = req.params.all();
+    console.log("Visit.create params=", params);
+
+    Visit.create(params, function(error, newVisit) {
+      if (error) return next(error);
```

Comment:
Moje główne doświadczenie jest z expressem, więc może być coś innego, ale czemu błąd jest przekazywany do dalszego requestu?
W expressie trzeci argument jest wykorzystywany do tworzenia interceptorów (np. inteceptor sprawdzający nasze dane autoryzacyjne może albo odrzucić request, albo zaakceptować te dane i przekazać request dalej do wykonania). Tu widzę odwrotną logikę, że jeśli wystąpi błąd to go przekazujemy dalej, a i tak i tak zwracamy 201.